hpx/runtime/parcelset/parcelhandler.hpp

```diff
@@ -194,10 +194,13 @@ namespace hpx { namespace parcelset
         ///                 id (if not already set).
         HPX_FORCEINLINE void put_parcel(parcel p)
         {
-            put_parcel(std::move(p), [=](
-                boost::system::error_code const& ec, parcel const & p) -> void {
-                    return invoke_write_handler(ec, p);
-                });
+            auto f =
+                [=](boost::system::error_code const& ec, parcel const& p) -> void
+                {
+                    invoke_write_handler(ec, p);
+                };
+
+            put_parcel(std::move(p), std::move(f));
```

Comment:
Is this one again about sequencing of captures and moves within a statement?

Reply:
No, this is about tricking the analysis tools used by Codacy not to complain about non-existent problems ;-)